### PR TITLE
Feature/accounts

### PR DESCRIPTION
**Pull request by sbellity**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "ui": false,
   "admin": "admin.html",
   "readme" : "readme.md",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (michaloo) [inline manifest.json]: That one should be `0.1.0` too, right?

File: server/compute.js [hunk 2]
```diff
@@ -20,6 +22,46 @@ function applyUtils(sandbox = {}) {
   sandbox._ = deepFreeze(lodash);
 }
 
+const buildPayload = (pld, pl = {}) => {
```
Comment (michaloo) [inline server/compute.js]: @KgaevskiT I'm not sure why we need to parse the `traits` arguments like this.
Couldn't we just pass the arguments directly to real `client.asUser().traits` or even expose the real method the the processor sandbox?

File: manifest.json (hunk 1 above)
Comment (KgaevskiT) [inline manifest.json]: yes you are right, I updated the version

File: server/compute.js (hunk 2 above)
Comment (KgaevskiT) [inline server/compute.js]: We want to call `client.asUser().traits` only once for optimization reasons. And this way we can compute the changes made, and call `client.asUser().traits` only if needed. Also we want to display the changes in the preview.

Comment (michaloo) [inline server/compute.js]: Yes, you are right, mostly the preview forces us to do all that data parsing here. Sorry about that.

File: server/server.js [hunk 3]
```diff
@@ -27,16 +27,16 @@ module.exports = function Server(app, options = {}) {
...
     },
     handlers: {
-      "user:update": (ctx, messages = []) => {
-        return Promise.all(messages.map(message => updateUser({ message }, {
-          ship: ctx.ship,
-          hull: ctx.client
-        })));
+      "user:update": ({ ship, client: hull }, messages = []) => {
+        return Promise.all(messages.map(message => {
+          // TODO: enable groupTraits option in the notifHandler
```
Comment (michaloo) [inline server/server.js]: I think you can use the `groupTraits` option but inside `userHandlerOptions` as in this example: https://github.com/hull/hull-node/#notifhandler

File: server/actions/notify-handler.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import updateUser from "../user-update";
+import { notifHandler } from "hull/lib/utils";
+
+export default function notify(hostSecret) {
+  return notifHandler({
+    hostSecret,
```
Comment (michaloo) [inline server/actions/notify-handler.js]: we should not need `hostSecret` here anymore.

File: server/server.js [hunk 5]
```diff
@@ -1,52 +1,20 @@
...
-
-  app.set("views", path.resolve(__dirname, "..", "views"));
-
-  app.get("/manifest.json", Manifest(__dirname));
-  app.get("/", Readme);
-  app.get("/readme", Readme);
+  app.post("/compute", ComputeHandler({ hostSecret, connector, Hull }));
 
+  if (options.devMode) app.use(devMode());
+  connector.setupApp(app);
```
Comment (michaloo) [inline server/server.js]: any reason for doing that here and not in `index.js`?

Comment (KgaevskiT) [inline server/server.js]: We need to define the `/compute` endpoint before applying middlewares for some reason..

**Pull request merged**